### PR TITLE
Updated model files to user @@SECRET macro; added sparce jms/wldf mod…

### PR DESCRIPTION
**Pull request by anpanigr**
a. Updated model files used in integration test to use @@SECRET instead of @@FILE macro
b. Added sparce jms/wldf model file 
c. Put the workaround to add a custom configuration/secret to resolve ConfigMap upgrade Issue
   ( This can be removed once @jshum2479 PR is merged into model-in-image branch) 

Initiated an external Jenkin Run. 

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-quicktest/1251/
(Known Intermittent Coherence Test Issue ) 
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-quicktest/1250/
( all Model in Image Test Passed )

### CODE REVIEW

File: integration-tests/src/test/resources/model-in-image/model.jms.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+resources:
```
Comment (markxnelson) [inline integration-tests/src/test/resources/model-in-image/model.jms.yaml:1]: copyright notice missing

File: integration-tests/src/test/resources/model-in-image/model.jrf.yaml [hunk 2]
```diff
@@ -1,12 +1,11 @@
 domainInfo:
```
Comment (markxnelson) [inline integration-tests/src/test/resources/model-in-image/model.jrf.yaml:1]: copyright notice missing

**Comment by jshum2479** [comment]
We still needs to wait for WDT new release before testing it.  The change in operator side is already merged.

File: integration-tests/src/test/resources/model-in-image/model.jrf.yaml (hunk 2 above)
Comment (anpanigr) [reply integration-tests/src/test/resources/model-in-image/model.jrf.yaml:4]: done

File: integration-tests/src/test/resources/model-in-image/model.jms.yaml (hunk 1 above)
Comment (anpanigr) [reply integration-tests/src/test/resources/model-in-image/model.jms.yaml:4]: done

File: integration-tests/src/test/resources/model-in-image/miisection-toaddin-domain-template.yaml [hunk 3]
```diff
@@ -1,5 +1,11 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# This file need to appended into kubernetes/samples/scripts/common/domain-template.yaml
+
+
   %MII_PREFIX%configuration:
-  # secrets: [ %DOMAIN_UID%-rcu-access ]
+    #%MII_PREFIX%  secrets: [ %WEBLOGIC_CREDENTIALS_SECRET_NAME% ]
```
Comment (rjeberhard) [inline integration-tests/src/test/resources/model-in-image/miisection-toaddin-domain-template.yaml:8]: Why would we generate commented out content?

File: integration-tests/src/test/resources/model-in-image/model.cluster.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+# A sparce model file to add a Configured Cluster 
+
+topology:
+    Cluster:
+        'cluster-2':
+    Server:
+        'config-managed-server-1':
+            Cluster: 'cluster-2'
+            ListenPort: '@@PROP:MANSVR_PORT@@'
+            NumOfRetriesBeforeMsiMode: 0
```
Comment (rjeberhard) [inline integration-tests/src/test/resources/model-in-image/model.cluster.yaml:13]: Are the MSI properties required?  I want to make sure that this isn't a workaround to a bug...

**Comment by markxnelson** [comment]
if ryan approves, i also approve

**Comment by rjeberhard** [comment]
@anpanigr, everything looks good here, but I'd like to understand items related to the two comments I made before approving.

File: integration-tests/src/test/resources/model-in-image/model.cluster.yaml (hunk 4 above)
Comment (anpanigr) [reply integration-tests/src/test/resources/model-in-image/model.cluster.yaml:13]: No this is not a  workaround. I had a sample model

Comment (rjeberhard) [reply integration-tests/src/test/resources/model-in-image/model.cluster.yaml:13]: Then please remove the attributes.

File: integration-tests/src/test/resources/model-in-image/miisection-toaddin-domain-template.yaml (hunk 3 above)
Comment (anpanigr) [reply integration-tests/src/test/resources/model-in-image/miisection-toaddin-domain-template.yaml]: That was a workaround for a WDT issue.  I removed the line on my latest commit as this is available in model-in-image

File: integration-tests/src/test/resources/model-in-image/model.cluster.yaml (hunk 4 above)
Comment (anpanigr) [reply integration-tests/src/test/resources/model-in-image/model.cluster.yaml]: Removed in my latest commit

**Pull request merged**